### ISSUE TITLE
Suppressions in shared/linked files do not work in SQ7.4

### ISSUE THREAD

**Issue by duncanp-sonar** [SonarSource/sonarlint-visualstudio#778]
### Description

Issues in linked files or files in shared projects that have been marked as fixed in SQ (FP, Won't Fix...) are not suppressed in the IDE.

### Repro steps

1. Create a solution with a normal project and a shared project
2. Bind the solution to a SQ project
3. Add a class file in the shared project that contains an issue
4. Analyse the project using the Scanner for MSBuild
5. Mark the issue as a FP in SQ
6. Re-open the solution in VS and open the shared file (NB it might be necessary to trigger a new analysis by editing the file due to a different bug)

### Expected behavior

The suppressed issue is not reported in the IDE.

### Actual behavior

The issue is still reported.

### TODO
Check whether the issue also occurs for versions of SQ prior to 7.2 (the format of the data returned by the issues API changed in 7.2)

### Known workarounds

None

### Related information

* SonarLint for VS Version: 4.7
* Visual Studio Version: VS2017 (not checked in VS2015 but presumably the same)
* SonarQube server version: 7.4

### Notes
SLVS issues a call like the following to fetch the list of issues:
`http://localhost:7400/api/issues/search?projects=slvs1&statuses=RESOLVED&p=1&ps=500`

SQ7.4 returns the following snippet for suppressions in linked/shared files:
```
    {
      "key": "AWgZ-nNTuwjFIkoqzRiF",
      "rule": "csharpsquid:S1144",
      "severity": "MAJOR",
      "component": "slvs1:shared/SharedProject1/ClassInSharedProject.cs",
      "project": "slvs1",
      "line": 11,
      "hash": "56a6afcaeeed577bf7e56b505661fd0b",
      "textRange": {
        "startLine": 11,
        "endLine": 14,
        "startOffset": 8,
        "endOffset": 9
      },
      "flows": [],
      "resolution": "FALSE-POSITIVE",
      "status": "RESOLVED",
      "message": "Remove the unused private method 'UnusedMethodInClassInSharedProject'.",
      "effort": "5min",
      "debt": "5min",
      "author": "anon@sonarsource.com",
      "tags": [ "unused" ],
      "creationDate": "2019-01-04T10:34:48+0000",
      "updateDate": "2019-01-04T18:19:40+0000",
      "type": "CODE_SMELL",
      "organization": "default-organization",
      "fromHotspot": false
    }

```
The problem is the format of the _component_ data, which isn't handled correctly by [this code](https://github.com/SonarSource/sonarlint-visualstudio/blob/9c5f245ec4113f2838d1f2178775e42b1b2de47f/src/SonarQube.Client/Api/Requests/V7_20/GetIssuesRequest.cs#L48) in SLVS.
